network/network.test.js

```diff
@@ -113,6 +113,19 @@ describe('useNetworkStatus', () => {
     expect(result.current.effectiveConnectionType).toEqual('4g');
   });
 
+  test('should update the effectiveConnectionType state when network get disabled', () => {
+    global.navigator.connection = {
+      ...ectStatusListeners,
+      effectiveType: '2g'
+    };
+
+    const { result } = renderHook(() => useNetworkStatus());
+    global.navigator.connection.effectiveType = 'network-unavailable';
```

Comment:
Is this `effectiveType` officially supported in a browser? I don't see it in [the spec](http://wicg.github.io/netinfo/) 🤔

Reply:
I don't believe so, I haven't encountered this before. Personally, I would prefer to stick closer to the spec

Reply:
This is the first time I've encountered `network-unavailable` and wasn't aware it was supported. Is it possible to learn where this value came from? :)

Reply:
Yeah, It's not supported. Sorry!, My bad. I've refactored the code.